data/mods/innawood/itemgroups/trash_and_debris.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "type": "item_group",
+    "id": "trash_forest",
+    "//": "Naturally occuring glass is considered to be from lightning strikes, maybe?",
+    "//2": "Naturally occuring metal is considered to be sufficiently pure nuggets",
+    "items": [
+      [ "rock", 80000 ],
+      [ "rock_flaking", 80000 ],
+      [ "stick", 80000 ],
+      [ "acorns", 80000 ],
+      [ "pinecone", 80000 ],
+      { "item": "withered", "prob": 80000, "count-min": 1, "count-max": 10 },
+      { "item": "plant_fibre", "prob": 80000, "count-min": 1, "count-max": 10 },
+      [ "straw_pile", 80000 ],
+      [ "tanbark", 80000 ],
+      [ "rock_large", 70000 ],
+      [ "ceramic_shard", 10000 ],
+      [ "pine_bough", 70000 ],
+      [ "splinter", 70000 ],
+      [ "hickory_root", 65000 ],
+      [ "hickory_nut", 65000 ],
+      [ "tinder", 50000 ],
+      { "item": "feather", "prob": 40000, "count-min": 1, "count-max": 10 },
+      [ "bone", 30000 ],
+      [ "flyer", -100 ],
```

Comment:
Are you sure negative probability will work?

Reply:
It does not, so I instead increased probability of the other spawns greatly. The items with negative values should probably be removed, come to think of it.

Reply:
Removed.